yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -137,7 +137,7 @@ export class Archiver implements ArchiveSource {
   }
 
   /**
-   * Fetches `L2BlockProcessed` and `ContractDeployment` logs from `nextL2BlockFromBlock` and processes them.
+   * Fetches logs from L1 contracts and processes them.
```

Comment:
Sneaked this change in. There is no longer any `ContractDeployment` log so this comment was outdated.